manifests/eap/access_log.pp

```diff
@@ -1,28 +1,25 @@
 # Defines access log pattern for 'default-host' virtual-server
 class lightblue::eap::access_log (
     $pattern = '\%h \%l \%u \%t \\\u0034\%r\\\u0034 \%s \%b \\\u0034\%{Referer}i\\\u0034 \\\u0034\%{User-Agent}i\\\u0034 \%D \%S',
-    $rotate  = false,
+    $rotate  = 'true',
     $prefix  = 'access_log',
-    $resolve_hosts = false
+    $resolve_hosts = 'false',
```

Comment:
So that <% if resolve_hosts and resolve_hosts != '' %> is true when resolve_hosts is false ('false').

To be honest, I don't follow how checking if variable is defined works in puppet. Sometimes it's !=undef, other times it's !=nil. I also remember seeing different behavior depending on how the the var is accessed (scope.lookupvar vs directly).